apps/ledger-live-mobile/src/screens/Swap/SubScreens/SelectCurrency.tsx

```diff
@@ -32,22 +43,35 @@ export function SelectCurrency({
       // @ts-expect-error navigation type is only partially declared
       navigation.navigate(ScreenName.SwapForm, { currency });
     },
-    [track, navigation],
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+    [],
   );
+
+  const RenderItem = useCallback(({ item }: { item: CryptoCurrency | TokenCurrency }) => {
```

Comment:
does renderItem need to be memoized in the callback? I usually just use `renderItem = (item) => <Row currency={item} ... />` So it looks to me like we could avoid the useCallback and eslint-disable.

If there is a need, `renderItem(onSelect)` could also be a function outside of the SelectCurrency component, which returns a function that takes the flatlist data item props, i.e. 
```
function renderItem(onSelect) {
  return ({currency}: { currency: CryptoCurrency | TokenCurrency }) => 
    <CurrencyRow currency={currency} onPress={onSelect} />
}
```
but I don't know if that's really necessary.